tests/wpt/servowpt.py

```diff
@@ -154,10 +154,23 @@ def run_tests(**kwargs):
 
     # Filter intermittents if that was specified on the command-line.
     if handler.unexpected_results and filter_intermittents_output:
-        all_filtered = filter_intermittents(
-            handler.unexpected_results,
-            filter_intermittents_output
-        )
+        unexpected_results = list(handler.unexpected_results)
+
+        print(80 * "=")
+        print(f"Rerunning {len(unexpected_results)} tests "
+              "with unexpected results to detect flaky tests.")
+        unexpected_results_tests = [result.path for result in unexpected_results]
+        kwargs["test_list"] = unexpected_results_tests
+        kwargs["include"] = unexpected_results_tests
+        kwargs["pause_after_test"] = False
+        wptrunner.run_tests(**kwargs)
```

Comment:
just to confirm, is it ok that we’re reusing the handler instance? it looks ok, since ServoHandler.suite_start happens to call reset_state, but it might be worth calling that explicitly here to avoid future bugs

Reply:
Yes, that's correct. I've added a call to `reset_state()` here and a comment explaining it.